lib/modules/dosomething/dosomething_user/dosomething_user_social_auth.inc

```diff
@@ -0,0 +1,98 @@
+<?php
+
+function dosomething_user_make_facebook_user($account_details) {
+  $edit = [
+    'mail' => $account_details['email'],
+    'name' => $account_details['email'],
```

Comment:
Oh never mind, I see what's happening below. Mind adding a inline comment here for clarity?